tests/rptest/tests/wasm_identity_test.py

```diff
@@ -216,6 +218,7 @@ def wasm_test_plan(self):
                        script=WasmTemplateRepository.IDENTITY_TRANSFORM)
         ]
 
+    @ignore  # https://github.com/redpanda-data/redpanda/issues/4061
```

Comment:
should this be `@ok_to_fail` which is now available? so it'd run, but the outcome would not contribute to CI success/failure.

Reply:
yep, I think that might have raced with my edit.  Changed this to ok_to_fail.